Services/Administration/OJS.Services.Administration.Business/IAdministrationOperationService.cs

```diff
@@ -5,7 +5,7 @@
 using System.Threading.Tasks;
 
 public interface IAdministrationOperationService<TEntity, in TId, TUpdateModel> : IService
-    where TEntity : class, IEntity<TId>
```

Comment:
And here

Reply:
Because UserProfile does not comes from IEntity<Tid>, therefore cannot be used in our workflow.